kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/instruction/ApplozicPermissions.java

```diff
@@ -18,6 +18,11 @@
     private LinearLayout snackBarLayout;
     private Activity activity;
 
+    public static final int REQUEST_STORAGE_MULTI_SELECT_GALLERY = 101;
```

Comment:
@shubhamtewari please check if same 101 is being used anywhere else? and if there is any possibility of conflict with other request codes

Reply:
an activity return code had the same value, but there would not have been any conflict since these are different codes (intent result vs request code).
However ive cahnged them to be in the 900s to avoid any future conflicts